server/src/utils.ts

```diff
@@ -7,7 +7,9 @@ import {
   ResponseMessage,
 } from "vscode-languageserver-protocol";
 import fs from "fs";
+import fsAsync from "fs/promises";
```

Comment:
In a later PR we might want to replace sync `fs` functions with their promise-based alternatives.